tests/integration/components/gleam-logo-test.js

```
@@ -0,0 +1,143 @@
+    assert.ok(canvas, 'Canvas element exists');
+    assert.strictEqual(canvas?.width, 400, 'Canvas width is set to base size for quality');
+    assert.strictEqual(canvas?.height, 400, 'Canvas height is set to base size for quality');
+    assert.strictEqual(canvas?.style.width, '100%', 'Canvas width style is set correctly');
+    assert.strictEqual(canvas?.style.height, '100%', 'Canvas height style is set correctly');
+    assert.strictEqual(canvas?.style.display, 'block', 'Canvas display style is set correctly');
+    assert.strictEqual(canvas?.style.maxWidth, '100%', 'Canvas max-width style is set correctly');
+  });
+
+  test('it handles hover state correctly', async function (assert) {
+    await render(hbs`<GleamLogo />`);
+    await settled();
```

ESLint

[error] Do not call `settled()` right after a test helper that already calls it internally.

(ember/no-settled-after-test-helper)